packages/mapviewer/tests/cypress/tests-e2e/importToolFile.cy.js

```diff
@@ -502,12 +503,28 @@ describe('The Import File Tool', () => {
             fixture: outOfBoundKMLFile,
         })
 
+        const wrongContentTypeHeadConfig = {
+            statusCode: 200,
+            headers: {
+                'Content-Type': 'text/plain',
+            },
+            fixture: validKMLFile,
```

Comment:
no need to give a fixture to a HEAD request (that's the whole purpose, don't give me the content, just the headers :wink: )